.github/workflows/master.yml

```diff
@@ -70,16 +70,18 @@ jobs:
     strategy:
       matrix:
         python-version: [3.6, 3.7]
-        include:
```

Comment:
I don't think this is necessary for now. We can do it when actually we need to add some more matrix.

Reply:
okay let's keep this for now.

Reply:
Let's close for now and reopen when we need.